NickvisionTubeConverter.GNOME/Program.cs

```diff
@@ -60,17 +62,15 @@ public Program(string[] args)
     /// <summary>
     /// Runs the program
     /// </summary>
-    /// <param name="args">Command-line arguments</param>
     /// <returns>Return code from Adw.Application.Run()</returns>
-    public int Run(string[] args)
+    public int Run()
     {
         try
         {
             SynchronizationContext.SetSynchronizationContext(new GLib.Internal.MainLoopSynchronizationContext());
-            var argc = args.Length + 1;
+            var argc = 1;
             var argv = new string[argc];
             argv[0] = "org.nickvision.tubeconverter";
-            args.CopyTo(argv, 1);
             return _application.Run(argc, argv);
```

Comment:
You could use `_application.RunWithSynchronizationContext` here

Reply:
RunWithSynchronizationContext doesn't accept args though, no?

Only in 0.5.0 preview it does.

Reply:
`_application.Run(argc, argv)` passes `org.nickvision.tubeconverter`...I thought that was needed for dbus stuff?

Reply:
I don't think so. ID is set in `Adw.Application.New`

Reply:
Ah ok. So then why did we ever pass them in the first place 😅

Reply:
maybe it was copied from Denaro or Tagger? 😄

Reply:
Ah makes sense...something from the early days 😂 we've come a long way in our apps that sometimes some parts feel like legacy code that should remain untouched hahaha